tarodb/sqlc/queries/assets.sql

```diff
@@ -418,6 +418,23 @@ FROM genesis_assets;
 SELECT * 
 FROM genesis_points;
 
+-- name: FetchGenesisID :one
+WITH target_point(genesis_id) AS (
+    SELECT genesis_id
+    FROM genesis_points
+    WHERE genesis_points.prev_out = @prev_out
+)
+SELECT gen_asset_id
+FROM genesis_assets
+WHERE (
+    (genesis_assets.genesis_point_id IN (SELECT genesis_id FROM target_point))
+    AND (genesis_assets.asset_id = @asset_id) AND
```

Comment:
For style consistency, how about putting this `AND` above to the right like the rest are below?